sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/HiveTableScanExec.scala

```diff
@@ -62,6 +62,8 @@ case class HiveTableScanExec(
 
   override def conf: SQLConf = sparkSession.sessionState.conf
 
+  override def nodeName: String = s"${super.nodeName}-${relation.tableMeta.qualifiedName}"
```

Comment:
`s"${super.nodeName}(${relation.tableMeta.qualifiedName})"` looks clearer to me, but up to you.

Reply:
I like this format. I added a space in between for better readability: `nodename (tablename)`

Reply:
(updated the screenshot in the PR description)